README.md

```diff
@@ -122,6 +122,14 @@ Then you can run the test suite with,
 If you disable flang support during configure, Fortran tests will be
 disabled and considered as passing the test.
 
+### Cleanning tests
+
+You can clean the test directory with,
+
+```bash
+   $ make cleantests
```

Comment:
Can we have just `make clean` do the cleaning instead of `make cleantests` ?

Reply:
Yes it will be more efficient. I will try to add my cleantests target to clean target.